workloads/templates/vm-creation.md

```diff
@@ -0,0 +1,106 @@
+# Virtual Machine Creation
+
+## Overview
+
+KubeVirt adds new types to the Kubernetes API to manage Virtual Machines and, in OpenShift, you can interact with these new resources via `oc` command as you would with any other Kubernetes API resource.
```

Comment:
Stroke this line - This is pretty redundant to other parts of the docs.

Please instead add, a not to long sentence along the lines of:

> OpenShift's template mechanism allows a user to create a set of objects from a template. KubeVirt is using this as a template mechanism for new OfflineVirtualMachines. The following chapter concentrates on the specific use around OfflineVirtualMachines, more informations on the templating mechanism can be [found in the OpenShift guide](https://docs.openshift.org/latest/dev_guide/templates.html).

Important points:
- Tell where Templates come from and what they do - link to the details
- Tell what we are using template for

Then (as you do) continue below with a detailed description.